pom.xml

```diff
@@ -143,12 +143,24 @@
                 <artifactId>apache-jena-libs</artifactId>
                 <version>${dependency.jena}</version>
                 <type>pom</type>
+                <exclusions>
```

Comment:
Bother.

That should be scope=test in Jena. Will fix in the post-release clearup.